libs/File2DProcessingTools/include/CSvgFileWriter.hpp

```diff
@@ -34,12 +35,31 @@ namespace File2DProcessingTools {
          */
         void write(std::string filepath, const CVectorGraphicsData &data, bool write_force) override;
 
+        void
+        writeMul(std::string filepath, const std::vector<CVectorGraphicsData> &data, bool write_force) override;
+
+
     private:
 
+        void writeBeginning(std::stringstream &stream);
+
+        static void writeEnd(std::stringstream &stream);
+
+        void writePoints(const File2DProcessingTools::CVectorGraphicsData &data,
+                         std::stringstream &stream, std::pair<double, double> max, std::pair<double, double> min);
+
         static bool isFileExists(const std::string &filepath, bool write_force);
 
         double scaling(unsigned int height, unsigned int width);
 
+        static void checkFile(const std::string &filepath, bool write_force);
```

Comment:
Add return value (bool), rename checkFile -> tryOpenFile

`bool tryOpenFile(const std::string &filepath, std::ofstream &fout, bool write_force)`

Usage:
```c++
std::ofstream fout;
if (!tryOpenFile(filepath, fout, write_force) {
    // Unreachable "return" because of exceptions, but left it here
    return;
}

fout << "I'm writing data...";
fout.close();
```